scss/components/_commercial.scss

```diff
@@ -1,27 +1,28 @@
 .commercial-data {
-  margin-top: -40px;
+  padding: 25px 0;
   width: $container-width;
-  height: 294px;
   border: 1px solid black;
-  background: #656d7a;
+  background: $grey;
 }
 .list-of-data {
-  display: flex;
-  color: #fafafa;
+  color: $white;
   height: 100%;
-  justify-content: space-between;
-  align-items: center;
+  padding-bottom: 215px;
+}
+.list-of-data li {
```

Comment:
What about?
```
.list-of-data {
  li {}
}
```
And use class instead of tag `li`

Reply:
Got it